backend/src/common/utils.ts

```diff
@@ -58,4 +56,13 @@ export const generateData = () => {
   return people;
 };
 
-console.log(generateData());
+const handler = async () => {
+  const app = await NestFactory.createApplicationContext(AppModule);
+  const data = generateData();
+
+  const personnelService = app.get(PersonnelService);
+  console.log(data);
+  return await personnelService.createPersonnel(data);
+};
+
+console.log(handler());
```

Comment:
?

Reply:
You have to run the makefile command and it only runs locally at this point. I just wanted to get an approval on this first. For seeding on dev/test we would only need to run once since the data persists, so I just figured I would tunnel in and run the makefile command.